fhir-audit/src/main/java/com/ibm/fhir/audit/logging/impl/WhcAuditCadfLogService.java

```diff
@@ -69,22 +69,22 @@
     private boolean isEnabled = false;
 
     private static final Map<String, Action> fhir2CadfMap = new HashMap<String, Action>() {
-
         private static final long serialVersionUID = 1L;
         {
             put("C", Action.create);
-            put("D", Action.delete);
-            put("U", Action.update);
             put("R", Action.read);
+            put("U", Action.update);
+            put("D", Action.delete);
```

Comment:
the ordering made me smile :)

Reply:
I did a quick review of our auditing stuff and I think it needs some love, but for now I contained myself to this simple change :-)